purefa/README.md

```diff
@@ -12,7 +12,7 @@ You can monitor multiple FlashArrays and aggregate these into a single dashboard
 
  - Agent v7.26.x+ to utilize OpenMetricsBaseCheckV2
  - Python 3
- - The Pure Storage Prometheus exporter is installed and running in a containerized environment. Refer to the [GitHub repo][1] for installation instructions.
+ - The Pure Storage OpenMetrics exporter is installed and running in a containerized environment. Refer to the [GitHub repo][1] for installation instructions.
```

Comment:
👋 For this setup do you envision users using a containerized agent as well? The reason why I'm asking is because installing the an extras check in a containerized agent isn't really straightforward. But if the intended setup involves just having the agent in the host/node and polling metrics from containers then it should be fine.

Reply:
I'd really like it to be that way! So I'd like to keep the option open to our customers. 

The idea would be for the datadog-agent containers running in k8s to directly query the OpenMetrics containers that we have an image for (linked in the readme) in the same cluster.

Reply:
Gotcha. I envision some difficulty with customers using the check with a containerized agent because the agent will need to be restarted after the check/integration installation step, which isn't really easy since the container will just get deleted and a new one will be created to take it's place. But that might be outside the scope of this PR.

Reply:
Agree, but will keep in mind and I'll probably reach out if any customers specifically require it. Also there's also other methods of ingestion that we are working on that could even bypass the need for this agent integration. (PureFA OTel Receiver)